.all-contributorsrc

```diff
@@ -69,7 +69,17 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "TomAFrench",
```

Comment:
I'm not sure why this is showing up here, should already be in master 🤔

Reply:
weird